docker-compose-local-relay.yml

```diff
@@ -28,7 +28,8 @@ services:
       --ws-external
       --rpc-methods=Unsafe
       --alice
-      --log="main,debug"
+      --wasmtime-instantiation-strategy=legacy-instance-reuse
+      --log="main,info,xcm=trace,xcm-executor=trace"
```

Comment:
notes:

1. We need line 31 to have the relay chain working on M1 machines
2. Line 2 is edited to match bob's flags, so that XCM-related logs are printed